### ISSUE TITLE
[Bug]: Map/List switch does not have a focus outline

### ISSUE THREAD

**Issue by nickvisut** [sfbrigade/support-sfusd#251]
### What happened?

While navigating the map page via keyboard, there is a focus outline around most navigable items with one important exception: the Map/List switch

Due to this, if a user attempts to navigate (e.g., by pressing the tab key) until they reach it, they will be unable to tell via visual cue that it is focused.

### What should have happened instead?

There ought to be a focus outline around the Map/List switch when the switch is focused (the entire thing, not "map" and "list" separately)

### Version

supportsfschools.org (Production)
